website/content/docs/connect/ca/vault.mdx

```diff
@@ -116,16 +116,25 @@ The configuration options are listed below.
 
 
 - `RootPKIPath` / `root_pki_path` (`string: <required>`) - The path to
-  a PKI secrets engine for the root certificate. If the path does not
+  a PKI secrets engine for the root certificate.
+
+  If the path does not
   exist, Consul will mount a new PKI secrets engine at the specified path with the
   `RootCertTTL` value as the root certificate's TTL. If the `RootCertTTL` is not set,
   a [`max_lease_ttl`](https://www.vaultproject.io/api/system/mounts#max_lease_ttl)
   of 87600 hours, or 10 years is applied by default as of Consul 1.11 and later. Prior to Consul 1.11, 
   the root certificate TTL was set to 8760 hour, or 1 year, and was not configurable. 
   The root certificate will expire at the end of the specified period. 
-  
+
   When WAN Federation is enabled, each secondary datacenter must use the same Vault cluster and share the same `root_pki_path`
-  with the primary datacenter. 
+  with the primary datacenter.
+
+  To use an intermediate certificate as the primary CA in Consul initialize the
+  `RootPKIPath` in Vault with a PEM bundle. The first certificate in the bundle
+  must be the intermediate certificate that Consul will use as the primary CA.
+  The last certificate in the bundle must be a root certificate. The bundle
+  must contain a valid chain, where each certificate is followed by the certificate
+  that authorized it.
```

Comment:
It might be a good idea to be more specific in calling out which endpoint(s) to use to set up the pki backend like this. Probably better to be overly detailed when we expect the backend to be set up a certain way.

Reply:
Ya, that's a good idea. It's a few steps, but I'll try and make it fit into this doc.  If it doesn't fit we might need to document it in a section below and link to it.

Reply:
I opened #12393 for these docs, so that it's easier to focus on the docs changes in isolation.